### ISSUE TITLE
`tilt down` outputs scary but harmless errors if no resource types exist for a group

### ISSUE THREAD

**Issue by milas** [tilt-dev/tilt#4610]
### Expected Behavior
* `tilt down` deletes all resources managed by Tilt without lots of scary errors making it seem like it failed

### Current Behavior
* If there are no resources for a group-version, an error is encountered (this is an odd scenario, see context) by some deep internal parts of the K8s client which are logged (via `klog`) and so appear in the CLI although they do not impact the actual cleanup
  ```
  E0603 14:52:57.741963   25291 memcache.go:196] couldn't get resource list for external.metrics.k8s.io/v1beta1: Got empty response for: external.metrics.k8s.io/v1beta1
  ```

### Context
This seems to be most prevalent with `external.metrics.k8s.io/v1beta1` (Google for `couldn't get resource list` and 99% of results will be for this) from the [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) because the external group exists but no external metric types are actually registered.

The specific error comes from `memcache.go`: https://github.com/kubernetes/kubernetes/blob/a4e5239bdc3d85f1f90c7811b03dc153d5121ffe/staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go#L212-L221

We'll likely need to just look for the magic string `Got empty response for` in the error and ignore it since that means `len(r.APIResources) == 0`.

**Comment by milas** [comment]
I'm actually fairly certain this is just benign (albeit confusing/annoying) logging coming from `klog` and doesn't actually interfere with tearing down resources. Once confirmed, I'll adjust this to simply suppress the message (we already filter out some other less-than-useful messages so this is straightforward).

**Comment by milas** [comment]
This is fixed in v0.20.7+
